tests/pro-archives/task.yaml

```diff
@@ -0,0 +1,22 @@
+summary: Chisel can fetch packages from Ubuntu Pro archives
+
+variants:
+  - noble
```

Comment:
Because in the chisel.yaml inside the chisel-releases/ directory, we only use the noble archive definition. Since this is meant to be a smoke test, it didn’t seem more fruitful to create chisel-releases/ directories for all variants.